docs/First-Installation.md

```diff
@@ -21,65 +21,58 @@ The simplest way to get most of these components is through an all-in-one kit su
-
-If you get the error `Illegal Instruction` when it boots or if it just boots into the terminal, you picked the wrong SD image or the image was corrupted on download or extraction.
-
-### Extract
-
-Once you have downloaded your SD card image you need to extract it using a program such as [7-Zip](http://www.7-zip.org/). You will extract the downloaded **.gz** file and the extracted file will be a **.img** file.
-
-To extract from the command line, you can type the following into a Terminal window, placing X with version you downloaded:
-
-`gunzip retropie-4.X.X-rpi2_rpi3.img.gz`
-
-### Install
-
-To install the RetroPie SD image on your MicroSD card. (You may need a MicroSD card reader to plug it into your computer) 
-
-1. For Windows you can use [Etcher](https://www.balena.io/etcher/), [Win32DiskImager](http://sourceforge.net/projects/win32diskimager/) or [Raspberry Pi Imager](https://www.raspberrypi.org/software/)
-2. For macOS you can use [Etcher](https://www.balena.io/etcher/), [Apple Pi Baker](https://www.tweaking4all.com/hardware/raspberry-pi/applepi-baker-v2/) or [Raspberry Pi Imager](https://www.raspberrypi.org/software/)
-3. For Linux you can use the `dd` command, [Etcher](https://www.balena.io/etcher/) or [Raspberry Pi Imager](https://www.raspberrypi.org/software/).
-
-See the official Raspberry Pi "WRITING AN IMAGE TO THE SD CARD" [instructions](https://www.raspberrypi.org/documentation/installation/installing-images/).
-
-**Note** RetroPie is built on top of Raspberry Pi OS Buster (a Linux based OS for the Raspberry Pi) and as such the partition on the SD card is EXT4 (a linux filesystem). This partition is is not visible on Windows systems, so the card will show up as a smaller size than usual and you won't be able to see everything on the card, but it is all there. You will be able to access the filesystem over the network as described in the transferring roms section below.
-
-If you're updating from a previous version of RetroPie see [**HERE**](Updating-RetroPie).
+### Preferred Method
+
+1. First, insert the MicroSD card into into your computer.
+2. Download the official [Raspberry Pi Imager](https://www.raspberrypi.org/software/), which allows downloading the different OS versions of RetroPie.
+3. Click the **Choose OS** button
+4. Select **Emulation and game OS**
+5. Select **RetroPie**.
+6. Select the image for the model of Raspberry Pi (RPI) that you have. For example, if you have a Raspberry Pi 4, select the RPI 4/400 option ([example](https://user-images.githubusercontent.com/540857/107868009-493a6400-6e4e-11eb-9272-7f45d569dc44.gif)) 
+   * If you don't know which model Raspberry Pi you have, there are a few methods to find out: 
+     * The [Raspberry Pi Wikipedia page](https://en.wikipedia.org/wiki/Raspberry_Pi#Connectors) has user-friendly graphics to help determine model by looking at the board itself 
+     * If you have SSH/commandline access, run `cat /proc/device-tree/model` and it will output your Pi's model
```

Comment:
**These nested list sections are probably going to break!**

I noticed it happens only if you switch from numbers to bullet points - MD has to have 3 spaces after the switch or it gets ignored, but that Material extension doesn't like 3 spaces. I'm leaving this formatted for MD for my sanity so I can do these edits and see the end result.

Reply:
We can fix those after. But you can't mix both list types due to the `mkdocs` extension for lists.

Reply:
@cmitu Do you mean the _mdx_truly_sane_lists_ extension? 

So it limits spaces from 4 to 2 for nested lists _and_ it messes with mixing ordered/unordered nested lists? Does it have an upside?

Reply:
Yes, it's easier to write without producing excessive white space between list items.